### PR TITLE
docs(web-modeler): add business rule task linking section and update …

### PR DESCRIPTION
**Pull request by nithinssabu**
…call activity linking section

Closes: https://github.com/camunda/web-modeler/issues/1567

## What is the purpose of the change

_Briefly describe the change you are making, this helps the reviewer by providing an overview._
Add a section about the business rule task linking feature and update the documentation for call activity linking

## Are there related marketing activities

_Are there any marketing activities related to the related feature or component? If so, please briefly describe them._
No

## When should this change go live?

_Does this change need to be released before a certain date or milestone? If so, when?_
As soon as possible

## PR Checklist

- [x] My changes apply to an already released version, and I have added them to the relevant `/versioned_docs` directory, or they are not for an already released version.
- [x] My changes apply to future versions, and I have added them to the main `/docs` directory, or they are not for future versions.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: docs/components/modeler/web-modeler/advanced-modeling/business-rule-task-linking.md
Comment (jfriedenstab) [inline docs/components/modeler/web-modeler/advanced-modeling/business-rule-task-linking.md:13]: I'm not sure about this, but I would highlight "DMN decision" because it's an entry of the dropdown.

```suggestion
3. Click the **Link** button to complete the linking process. In the properties panel, the value **DMN decision** is chosen for the **Implementation** property and the Decision ID of the decision you chose to link is automatically copied to the **Called decision** section.
```

File: docs/components/modeler/web-modeler/advanced-modeling/business-rule-task-linking.md
Comment (jfriedenstab) [inline docs/components/modeler/web-modeler/advanced-modeling/business-rule-task-linking.md:23]: Same here
```suggestion
You may also enter the Decision ID directly in the **Called decision** section in the properties panel after selecting **DMN decision** for the **Implementation**.
```

File: versioned_docs/version-8.1/components/modeler/web-modeler/advanced-modeling/business-rule-task-linking.md
Comment (jfriedenstab) [inline versioned_docs/version-8.1/components/modeler/web-modeler/advanced-modeling/business-rule-task-linking.md:23]:
```suggestion
You may also enter the Decision ID directly in the **Called decision** section in the properties panel after selecting **DMN decision** for the **Implementation**.
```

File: versioned_docs/version-8.1/components/modeler/web-modeler/advanced-modeling/business-rule-task-linking.md
Comment (jfriedenstab) [inline versioned_docs/version-8.1/components/modeler/web-modeler/advanced-modeling/business-rule-task-linking.md:13]:
```suggestion
3. Click the **Link** button to complete the linking process. In the properties panel, the value **DMN decision** is chosen for the **Implementation** property and the Decision ID of the decision you chose to link is automatically copied to the **Called decision** section.
```

**Review by jfriedenstab — CHANGES_REQUESTED** [review]
Thanks a lot for taking care of this!

Apart from the comments below:
- In `brt_linked.png` there's "Order discount" (lowercase "d") and "Order Discount" (uppercase "D"). I think it would be nice to use the same spelling.
- It would be nice to fix the [issue with the missing bullet points](https://github.com/camunda/web-modeler/issues/3693) in the overlay and update the screenshots again.

**Review by jfriedenstab — CHANGES_REQUESTED** [review]
Just two small issues left:
* In `linked.png` for Call Activity Linking, there's a small typo now: "invetory" instead of "inventory".
* The same image is pretty big now and takes up the whole width of the page; I'm not sure if that was intended.

**Comment by nithinssabu** [comment]
> Just two small issues left:

Sorry about this!

**Review by jfriedenstab — DISMISSED** [review]
Looks good to me now :)

**Comment by nithinssabu** [comment]
> Looks good to me now :)

Thanks :) I will wait for the approval from @christinaausley before merging this.

File: docs/components/modeler/web-modeler/advanced-modeling/business-rule-task-linking.md
Comment (christinaausley) [inline docs/components/modeler/web-modeler/advanced-modeling/business-rule-task-linking.md:30]: Is it worth briefly mentioning how to do this here?

**Review by christinaausley — APPROVED** [review]
Technical review in latest commit, and one non-blocking comment for clarification 👍

File: docs/components/modeler/web-modeler/advanced-modeling/business-rule-task-linking.md
Comment (nithinssabu) [reply docs/components/modeler/web-modeler/advanced-modeling/business-rule-task-linking.md:30]: I would say, no. This is advanced modeling and by the time a user reads this section, they should already know how to deploy a diagram.

**Pull request merged**
